simplq/src/components/common/Loading/Loading.jsx

```diff
@@ -21,3 +22,15 @@ export default ({ children, actionStatus, isLoading }) => {
     </div>
   );
 };
+
+Loading.defaultProps = {
+  isLoading: undefined,
+  actionStatus: undefined,
+};
+
+Loading.propTypes = {
+  isLoading: PropTypes.bool,
+  actionStatus: PropTypes.oneOf(['pending', 'fulfilled', 'rejected']),
```

Comment:
Let us also add `children: PropTypes.element.isRequired`?

Reply:
If we do that, the component can't be used on its own. E.g., dispatch joinQueue and bring modal spiner to the front until action is fulfilled.  Do we want to that?

Reply:
Ok, let's keep it as is. 👍